substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/Target_jdk_jfr_internal_JVM.java

```diff
@@ -339,4 +370,39 @@ public void emitOldObjectSamples(long cutoff, boolean emitAll, boolean skipBFS)
     public boolean shouldRotateDisk() {
         return SubstrateJVM.get().shouldRotateDisk();
     }
+
+    /** See {@link JVM#flush}. */
+    @Substitute
+    @TargetElement(onlyWith = JDK14OrLater.class) //
+    public void flush() {
+        // Temporarily do nothing. This is used for JFR streaming.
```

Comment:
Can we throw exceptions for all methods that are only used for JFR streaming? Or are those methods always called, like `retransformClasses`.

Reply:
These methods are similar to `retransformClasses` and are always called.